modules/operator-deploy-infrastructure.adoc

```diff
@@ -97,42 +97,77 @@ Use the following procedure to create a project with node selector and toleratio
 
 [NOTE]
 ====
-If you have already deployed {productname} using the Operator, remove the installed Operator and any specific namespaces that you created for the deployment. 
+The following procedure can also be completed by removing the installed {productname} Operator and the namespace, or namespaces, used when creating the deployment. Users can then create a new resource with the following annotation.
 ====
 
 .Procedure
 
-. Create a project resource, specifying a node selector and toleration. For example:
+. Enter the following command to edit the namespace where {productname} is deployed, and the following annotation: 
 +
-.quay-registry.yaml
 [source,terminal]
 ----
+$ oc edit project <project_name>
```

Comment:
That won't work,  when use this way to add new annotation to exiting OCP Project, will hit error "**metadata.annotations[openshift.io/node-selector]: Invalid value: "node-role.kubernetes.io/infra=": field is immutable, try updating the namespace**", should use oc annotate command, see following example:

```
oc annotate namespace quay openshift.io/node-selector='node-role.kubernetes.io/infra='
namespace/quay annotated
```